nuitka/optimizations/OptimizeBuiltinCalls.py

```diff
@@ -911,6 +914,114 @@ def wrapEvalBuiltin(source, globals_arg, locals_arg, source_ref):
     )
 
 
+def max_extractor(node):
+    @calledWithBuiltinArgumentNamesDecorator
+    def wrapMaxBuiltin(source, source_ref):
```

Comment:
PyLint is complaining due to the unused variable `source`. However, it is required in the Paramer Specs to pass argument in the wrap function:
https://github.com/Nuitka/Nuitka/blob/45d791faedbcba5daf7b95412314d4f61a02f216/nuitka/specs/BuiltinParameterSpecs.py#L577

Should I supress the Pylint error?